apps-rendering/src/components/liveblogHeader.tsx

```diff
@@ -46,9 +45,12 @@ const LiveblogHeader: FC<Props> = ({ item }) => {
 			>
 				<Columns collapseUntil="desktop">
 					<Column span={3}>
-						<time css={timestampStyles}>
-							TODO: Updated timestamp
-						</time>
+						<Hide below="desktop">
+							<LiveDateline
+								date={item.publishDate}
```

Comment:
I think we're going to want to use some version of the "last modified" date for this. `publishDate` is when the piece was first published, which is likely to be some hours before it was actually last updated.

We don't currently have this on `Item` so I think we have two options:

1. Duplicate the logic that @AshCorr has used in #3988 to pull last modified from the most recent live block.
2. Parse `content.fields.lastModified` into a new `lastModifiedDate` on `Item` (`item.ts`) and use that.

Reply:
I did look at this, but the `publishDate` does get updated when the content does and using it keeps it in sync locally with the live version

Reply:
As discussed offline, I think you're right. `publishDate` is _not_ first published as I originally thought, it's actually the `webPublicationDate`, which is slightly different. For anyone else reading this, we're looking into how exactly that field gets updated, but for now I think we're ok to merge.